datavirt64/dynamicvdb-datafederation/mysql/customer-schema.sql

```diff
@@ -0,0 +1,199 @@
+DROP TABLE IF EXISTS status;
+DROP TABLE IF EXISTS h2_stock_mat;
+DROP TABLE IF EXISTS mat_stock_staging;
+DROP TABLE IF EXISTS CUSTOMER;
+DROP TABLE IF EXISTS ACCOUNT;
+DROP TABLE IF EXISTS PRODUCT;
+DROP TABLE IF EXISTS HOLDINGS;
+DROP TABLE IF EXISTS HT_ProductInfo;
+
+CREATE TABLE status
+(
+  VDBName varchar(50) not null,
+  VDBVersion integer not null,
+  SchemaName varchar(50) not null,
+  Name varchar(256) not null,
+  TargetSchemaName varchar(50),
+  TargetName varchar(256) not null,
+  Valid boolean not null,
+  LoadState varchar(25) not null,
+  Cardinality long,
+  Updated timestamp not null,
+  LoadNumber long not null,
+  PRIMARY KEY (VDBName, VDBVersion, SchemaName, Name)
+);
+
+CREATE TABLE h2_stock_mat
+(
+   product_id integer,
+   SYMBOL varchar(16),
+   PRICE decimal(20,4),
+   COMPANY_NAME varchar(256)            
+);
+
+CREATE TABLE mat_stock_staging
+(
+   product_id integer,
+   SYMBOL varchar(16),
+   PRICE decimal(20,4),
+   COMPANY_NAME varchar(256)            
+);
+
+CREATE TABLE CUSTOMER
+(
+   SSN char(10),
+   FIRSTNAME varchar(64),
+   LASTNAME varchar(64),
+   ST_ADDRESS varchar(256),
+   APT_NUMBER varchar(32),
+   CITY varchar(64),
+   STATE varchar(32),
+   ZIPCODE varchar(10),
+   PHONE varchar(15),
+   CONSTRAINT CUSTOMER_PK PRIMARY KEY(SSN)
+);
+
+
+CREATE TABLE ACCOUNT
+(
+   ACCOUNT_ID integer,
+   SSN char(10),
+   STATUS char(10),
+   TYPE char(10),
+   DATEOPENED timestamp,
+   DATECLOSED timestamp,
```

Comment:
It's just hunch. But according to setting of **sql_mode**, there could arise error:
ERROR 1067 (42000) at line 57: Invalid default value for 'DATECLOSED'
This is caused by "NO_ZERO_IN_DATE,NO_ZERO_DATE"
mysql> show variables like 'sql_mode' ;
The setting is default in newer MySQL... https://stackoverflow.com/questions/36882149/error-1067-42000-invalid-default-value-for-created-at